example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -1419,7 +1419,7 @@ describe('The S3 Ingest Granules workflow', () => {
         expect(definition.Comment).toEqual('Ingest Granule');
 
         // definition has all the states' information
-        expect(Object.keys(definition.States).length).toBe(14);
+        expect(Object.keys(definition.States).length).toBe(15);
```

Comment:
Just for posterity - this update was due to the modification to the template that included the message source?